bin/resalloc-check-vm-ip

```diff
@@ -31,4 +31,4 @@ test -n "$RESALLOC_RESOURCE_DATA"
 # we only put IP out in spawning script, nothing else
 set -- $(echo "$RESALLOC_RESOURCE_DATA" | base64 --decode)
 IP=$1
-ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout=10 "root@$IP" true
+ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout=10 "${SSH_USER-root}@$IP" true
```

Comment:
I am currently experimenting with `c8s` workers and ssh through `centos` user. The auto switch from `root` to `centos` user which is done by resalloc on first ssh failure is working for me.

Reply:
Yes, that's the one probably, as it automatically detects the ssh user.

Reply:
That's unfortunately a different script (not useable for periodic checks). I'm moving it into the `resalloc-server` package, too, though: #147